### PR TITLE
dracula-theme 1.2.5 (new formula)

### PR DESCRIPTION
**Pull request by benjamreynolds**

### CODE REVIEW

File: Formula/dracula-theme.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class DraculaTheme < EmacsFormula
+  desc "A dark color theme available for a number of editors."
+  homepage "https://github.com/dracula/emacs"
+  url "https://github.com/dracula/emacs/archive/v1.2.5.tar.gz"
+  sha256 "bc78039c90d5d27f6b1ccb7b32fabb3ecbf0d485421bc63ae5d5d6836b9780b4"
+  head "https://github.com/dracula/emacs.git"
+
+  bottle :unneed
```
Comment (dunn) [inline Formula/dracula-theme.rb]: This should be `bottle :disable`.

File: Formula/dracula-theme.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class DraculaTheme < EmacsFormula
```
Comment (dunn) [inline Formula/dracula-theme.rb]: For themes, I've been using the naming scheme `<name>-emacs` ([`dunn/emacs/ahungry-emacs.rb`](https://github.com/dunn/homebrew-emacs/blob/master/Formula/ahungry-emacs.rb), [`dunn/emacs/solarized-emacs.rb`](https://github.com/dunn/homebrew-emacs/blob/master/Formula/solarized-emacs.rb)); so could you rename this to `dracula-emacs.rb`?

File: Formula/dracula-theme.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  url "https://github.com/dracula/emacs/archive/v1.2.5.tar.gz"
+  sha256 "bc78039c90d5d27f6b1ccb7b32fabb3ecbf0d485421bc63ae5d5d6836b9780b4"
+  head "https://github.com/dracula/emacs.git"
+
+  bottle :unneed
+
+  depends_on :emacs => "24.1"
+
+  def install
+    elisp.install "dracula-theme.el", "dracula-theme.elc"
```
Comment (dunn) [inline Formula/dracula-theme.rb]: You'll need `byte_compile "dracula-theme.el"` above this line  if you want the `.elc`.

Comment (benjamreynolds) [inline Formula/dracula-theme.rb]: I'll  actually just take out the `.elc` I dont think its necessary in this case.

File: Formula/dracula-emacs.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class DraculaEmacs < EmacsFormula
+  desc "A dark color theme available for a number of editors."
```
Comment (dunn) [inline Formula/dracula-emacs.rb]: From Travis:
```
dunn/emacs/dracula-emacs:
  * Description shouldn't start with an indefinite article (A)

**Comment by dunn** [comment]
The test is also  failing, though I'm not sure why:
```
==> emacs -Q --batch -l /tmp/dracula-emacs-test-20170320-7405-1l2ct5h/test.el
Unable to find theme file for `dracula'
Error: dunn/emacs/dracula-emacs: failed
<0> expected but was
<255>.

**Comment by benjamreynolds** [comment]
Fixed the formula description error, Ill look into the other tomorrow. Its certainly puzzling why its failing.

**Comment by benjamreynolds** [comment]
Ok @dunn, This is in pretty good shape now, but Travis is still failing because the repository is not notable enough. Is there a way to circumvent that test?

**Comment by benjamreynolds** [comment]
The annoying bit is that the [parent repository](https://github.com/dracula/dracula-theme) that holds all the individual themes as submodules more than meets the notability requirements.

**Comment by dunn** [comment]
No worries.  Merged as 8d7b04b2d47a6591826e47769db49513835d0bc4, thanks!

**Pull request closed without merging**